src/tox.ini

```diff
@@ -273,44 +273,45 @@ deps = ruff
-#   3498        I001    [*] Import block is un-sorted or un-formatted
-#   2146        F401    [*] `.PyPolyBoRi.Monomial` imported but unused
-#   1964        E741    [ ] Ambiguous variable name: `I`
-#   1676        F821    [ ] Undefined name `AA`
-#   1513        PLR0912 [ ] Too many branches (102 > 12)
-#   1159        PLR0913 [ ] Too many arguments in function definition (10 > 5)
-#    815        E402    [ ] Module level import not at top of file
-#    671        PLR0915 [ ] Too many statements (100 > 50)
-#    483        PLW2901 [ ] Outer `for` loop variable `ext` overwritten by inner `for` loop target
-#    433        PLR5501 [*] Use `elif` instead of `else` then `if`, to reduce indentation
-#    428        PLR0911 [ ] Too many return statements (10 > 6)
-#    404        E731    [*] Do not assign a `lambda` expression, use a `def`
-#    351        F405    [ ] `ComplexField` may be undefined, or defined from star imports
-#    306        PLR1714 [*] Consider merging multiple comparisons. Use a `set` if the elements are hashable.
-#    236        F403    [ ] `from .abelian_gps.all import *` used; unable to detect undefined names
-#    116        PLR0402 [*] Use `from matplotlib import cm` in lieu of alias
-#    111        PLW0603 [ ] Using the global statement to update `AA_0` is discouraged
-#     78        F841    [*] Local variable `B` is assigned to but never used
-#     48        PLW0602 [ ] Using global for `D` but no assignment is done
-#     33        PLR1711 [*] Useless `return` statement at end of function
-#     24        E714    [*] Test for object identity should be `is not`
-#     20        PLR1701 [*] Merge `isinstance` calls
-#     17        PLW3301 [ ] Nested `max` calls can be flattened
-#     16        PLW1510 [*] `subprocess.run` without explicit `check` argument
-#     14        E721    [ ] Do not compare types, use `isinstance()`
-#     14        PLW0120 [*] `else` clause on loop without a `break` statement; remove the `else` and dedent its contents
-#     12        F811    [*] Redefinition of unused `CompleteDiscreteValuationRings` from line 49
-#      8        PLC0414 [*] Import alias does not rename original package
-#      7        E743    [ ] Ambiguous function name: `I`
-#      7        PLR0124 [ ] Name compared with itself, consider replacing `a == a`
-#      5        PLW0127 [ ] Self-assignment of variable `a`
-#      4        PLW1508 [ ] Invalid type for environment variable default; expected `str` or `None`
-#      3        PLC3002 [ ] Lambda expression called directly. Execute the expression inline instead.
-#      2        E742    [ ] Ambiguous class name: `I`
-#      2        PLE0302 [ ] The special method `__len__` expects 1 parameter, 3 were given
-#      1        F402    [ ] Import `factor` from line 259 shadowed by loop variable
-#      1        PLC0208 [*] Use a sequence type instead of a `set` when iterating over values
+# 10589	PLC0415	[ ] import-outside-top-level
```

Comment:
I updated this comment while I was touching this file. Looks like that hasn't been done in a while.